app/src/main/java/com/sdpteam/connectout/event/EventDataManager.java

```diff
@@ -2,7 +2,9 @@
 
 import androidx.lifecycle.LiveData;
 
-public interface EventDataManager {
+import com.sdpteam.connectout.mapList.MapListModelManager;
+
+public interface EventDataManager extends MapListModelManager {
```

Comment:
Move all that inside `MapListModelManager` which as said before is going to be renamed to `EventRepository`. Don't forget to change into `CompletableFuture` so the model has no concepts related to android UI.